extension/src/market/market-product.ts

```diff
@@ -0,0 +1,67 @@
+// Generated by Copilot to reflect https://json-schema.axonivy.com/market/10.0.0/product.json
+export interface MarketProduct {
+  $schema?: string;
+  installers: Installer[];
+  minimumIvyVersion?: string;
+}
```

Comment:
This is generated code only:
I think we should generate it as on all other editor-protocols ... package action which use the json-schema-tojson generator.